scripts/legislation/africa/tanzania_english.sh

```diff
@@ -1250,6 +1370,6 @@ function preprocess_state_and_language_input_file {
     apply_common_transformations_to_stdin "$language" | \
     remove_all_text_after_last_article | \
     amend_errors_in_articles | \
-    amend_errors_in_headers | \
-    remove_and_reinsert_article_titles
+    amend_errors_in_headers 
+    #remove_and_reinsert_article_titles
```

Comment:
Did you intend to comment out this line? All of the article titles have been removed in this revision.

Reply:
No, thank you for pointing that out.